lib/engines/content_object_store/test/integration/content_block_editions_test.rb

```diff
@@ -24,18 +24,19 @@ class ContentBlockEditionsTest < ActionDispatch::IntegrationTest
     document_title = "Some Title"
     block_type = "block_type"
     details = {
-      foo: "Foo text",
-      bar: "Bar text",
+      "foo" => "Foo text",
+      "bar" => "Bar text",
     }
 
     ContentObjectStore::SchemaService.expects(:schema_for_block_type).with(block_type).returns(schema)
-    ContentObjectStore::ContentBlockEdition.expects(:create!).with do |args|
-      args.to_h == {
-        document_title:,
-        block_type:,
-        details:,
-      }.with_indifferent_access
-    end
+
+    initial_document_count = ContentObjectStore::ContentBlockDocument.count
+    initial_edition_count = ContentObjectStore::ContentBlockEdition.count
+
+    # This UUID is created by the database so instead of loading the record
+    # we stub the initial creation so we know what UUID to check for.
+    ContentObjectStore::ContentBlockEdition.any_instance.stubs(:create_random_id)
+      .returns("49453854-d8fd-41da-ad4c-f99dbac601c3")
```

Comment:
Might be better if we record the UUID as a variable to save repeating ourselves later

Reply:
Yup good thinking